meta/heart-bits/contao-backend-theme-bundle/de.yml

```diff
@@ -1,7 +1,2 @@
 de:
-    title: heart-bits Backend-Theme
-    description: >
-        Backend-Theme mit einigen Anpassungen für das Corporate Design von heart-bits
-    keywords:
-        - back end
-        - theme
+    dependency: true
```

Comment:
I think you can still leave the title and description, as this will be shown in the Contao Manager, when installed.

Reply:
Thanks for clarifying, @fritzmg. Thought the bundle wouldn't even be displayed.